sklearn/ensemble/_gb.py

```diff
@@ -1637,3 +1648,10 @@ def apply(self, X):
         leaves = super().apply(X)
         leaves = leaves.reshape(X.shape[0], self.estimators_.shape[0])
         return leaves
+
+    # FIXME: to be removed in 0.26
+    @deprecated("Attribute n_classes_ was deprecated in version 0.24 and "
```

Comment:
```suggestion
    @deprecated("Attribute n_classes_ was deprecated in version 0.24 and "  # type: ignore
```

We need to put this comment to avoid `mypy` to complain here. This kind of new.

Reply:
You might need to rewrap the lien to count for the 80 characters.

Reply:
I still get some errors. I'm not sure how to move forward.